src/keboola_mcp_server/tools/data_apps.py

```diff
@@ -60,6 +60,9 @@ def add_data_app_tools(mcp: FastMCP) -> None:
 # LLM agent can still understand the type of the data app even if it is different from the known types
 SafeType = Union[Type, str]
 
+# Type of the authentication used in the data app
+AuthenticationType = Literal['no-auth', 'basic-auth', 'default']
```

Comment:
`'default'` seems a little redundant, the agent can just leave the field empty to keep existing config